ui/app/models/pki/issuer.js

```diff
@@ -42,6 +48,59 @@ export default class PkiIssuerModel extends PkiCertificateBaseModel {
   })
   uriSans;
 
+  @attr('string') issuerName;
+
+  @attr({
+    label: 'Leaf notAfter behavior',
+    subText:
+      'What happens when a leaf certificate is issued, but its NotAfter field (and therefore its expiry date) exceeds that of this issuer.',
+    dockLink:
```

Comment:
I don't see this doc link in your screenshot?

Reply:
Another good catch! Apparently I was thinking of linking fishing docks are something :smile: